instruqt-tracks/vault-advanced-data-protection-with-tokenization/enable-the-transform-secret-engine/setup-kubernetes

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+#fix path
+echo "export PATH=/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin" >> /root/.bashrc
+echo "alias k='kubectl'" >> ~/.bashrc
+#k8s setup
+echo "Setting up K8s..."
+until [ -f /opt/instruqt/bootstrap/host-bootstrap-completed ]
+do
+    sleep 1
+done
+echo "source /usr/share/bash-completion/bash_completion" >> /root/.bashrc
+echo "complete -F __start_kubectl k" >> /root/.bashrc
+echo "export VAULT_ADDR=http://127.0.0.1:8200" >> /root/.bashrc
+/usr/bin/start.sh
+echo "K8s setup complete"
+
+
+curl -fsSL https://apt.releases.hashicorp.com/gpg | apt-key add -
+add-apt-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main"
+
+apt update -y
+apt install -y kubectl 
+apt install -y consul 
+apt install -y vault
+
+git clone https://github.com/Andrew-Klaas/vault-go-demo-tokenization.git
+git clone https://github.com/Andrew-Klaas/instruqt-tokenization-lab.git 
```

Comment:
Please copy all the content from this repo into one in the hashicorp organization. I suggest using hashicorp/vault-adp-workshop-assets (if the content from this repo and the one above can be combined). Then clone from that.

Reply:
Fixed the links to point to the included assets in this repo.